resources/todomvc/big-dom-generator/javascript-web-components/generated-variables.css

```diff
@@ -0,0 +1,50 @@
+:root {
+	--complex-accent-color-default: rgba(140, 140, 140, 0.5);
+	--complex-border-bottom-color-default: rgba(140, 140, 140, 0.5);
+	--complex-border-color-default: rgba(140, 140, 140, 0.5);
+	--complex-border-left-color-default: rgba(140, 140, 140, 0.5);
+	--complex-border-right-color-default: rgba(140, 140, 140, 0.5);
+	--complex-border-top-color-default: rgba(140, 140, 140, 0.5);
+	--complex-column-rule-color-default: rgba(140, 140, 140, 0.5);
+	--complex-outline-color-default: rgba(140, 140, 140, 0.5);
```

Comment:
A lot of these variables have the same value, just different name, it this on purpose?
Are we just trying to assign as many variables as possible?

Reply:
The scalability challenge is to test variable resolution in the presence of many inherited variables, so we didn't put a lot of attention on the values, as long as they didn't disrupt the todoMVC style. That said, we can modify the values to make each variable unique.

What was not intended was to have each list item resolve 8 additional css styles, so I changed it so each `li` and `li > div` elements resolve 2 more styles (like in the other complex versions), and increased the number of generated constructable stylesheets.

Reply:
Thanks for the explanation!